tpm/tpm.go

```diff
@@ -904,6 +904,61 @@ func ReadEKCert(rw io.ReadWriter, ownAuth digest) ([]byte, error) {
 	return ekbuf, nil
 }
 
+// NVDefineSpace defines space in NVRAM for the indicated index.
+// Definition include the access requirements for writing and reading the area
```

Comment:
tiny nit: I'd prefer to add the definite article here: "Definition include" -> "The definition includes".

Also, please end the comment with punctuation: in this case, a period.